examples/minimal_pub_sub/src/minimal_subscriber.rs

```diff
@@ -6,6 +6,7 @@ fn main() -> Result<(), Error> {
     let context = rclrs::Context::new(env::args())?;
 
     let node = rclrs::create_node(&context, "minimal_subscriber")?;
+    let inner_node = node.clone();
```

Comment:
I just removed all the changes in this file since they weren't meant to land in the final code anyway https://github.com/ros2-rust/ros2_rust/pull/325/commits/ed583f67d44e382015766a325041b8b5389dfd38